cmd/ndm_daemonset/controller/disk_to_device_convertor.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2019 The OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controller
+
+import (
+	"github.com/openebs/node-disk-manager/pkg/udev"
+	"strings"
+)
+
+func (c *Controller) NewDeviceInfoFromDiskInfo(diskDetails *DiskInfo) *DeviceInfo {
```

Comment:
Yet to go through this code.. but who will invoke or create the "Device" resource? ndm operator or ndm daemon set?

Reply:
Assuming .. it is NDM DS

Reply:
NDM-Daemon set would create Device. Device and disk CR creation would happen w.r.t udev addEvent.